src/test/groovy/com/jmeter/websocket/plugin/JmeterAbstractSpec.groovy

```diff
@@ -0,0 +1,16 @@
+package com.jmeter.websocket.plugin
+
+import spock.lang.Specification
+
+import static org.apache.jmeter.util.JMeterUtils.initLocale
+import static org.apache.jmeter.util.JMeterUtils.loadJMeterProperties
+import static org.apache.jmeter.util.JMeterUtils.setJMeterHome
+
+abstract class JmeterAbstractSpec extends Specification {
+    def setupSpec() {
```

Comment:
I would not rely on Spock source code. That code might be done many years ago before such best practice rule was introduced.